### DISCUSSION TITLE
Fan speed during defrost

### DISCUSSION THREAD

**Discussion by adasek-dev** [veista/nilan#103] [Q&A]
Hi, Thanks in advance for your great work. I have a Combi 302 Polar unit and I would like to set the supply fan to 0 when the defrost is running. I think I saw someone have the option to set the defrost ventilation level. I don't have this option in HA. Is it possible to add it? If I manually set the HA fan level supply to 0, it returns to the original value within a few seconds. Can it be solved somehow? Thank you for your help.

**Comment by veista** [comment]
Once again, I can not help unless you provide me the information mentioned in the readme. Modbus Version, Device Type - as Shown in the Integration, Device Version - as Shown in the Integration.

**Reply by adasek-dev** [reply]
Modbus version = 22, Device type = Combi 302.

**Comment by veista** [comment]
Do you have such an option via your HMI panel?

**Reply by adasek-dev** [reply]
No, I don't have this option via the HMI panel. That's why I asked if it was possible to set this somehow, because when the defrost is in progress, too cold air blows into my house. I would like to limit this somehow. For example, switch off the supply fan via node-red during defrost. I can set that, but the previous value is set within a second.

**Reply by veista** [reply]
Then combi 302 does not have that function. You can try writing to that register, but I don't think that will do anything.

**Reply by adasek-dev** [reply]
OK, thank you for your time.
